README.md

```diff
@@ -1 +1,117 @@
-# awesome-book
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h2><b>Welcome to our Project</b></h2>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+**[Awesome Books]** is an application that shows a books list. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 
+- **[HTML5]**
+- **[Javascript]**
+
+### Key Features <a name="key-features"></a>
+
+- **[Dynamic Content]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+-Clone the repository.
+-Create a directory in your device.
+
+### Prerequisites
+
+In order to run this project you need:
+-A code editor like VS code
+-Internet connection
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+  git clone https://github.com/jecode93/awesome-book.git
+ 
+  cd awesome-book
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jean Emmanuel Cadet**
+
+- GitHub: [@jecode93](https://github.com/jecode93)
+- Twitter: [@jecode93](https://twitter.com/jecode93)
+- LinkedIn: [jean-emmanuel-cadet](https://www.linkedin.com/in/jean-emmanuel-cadet/)
+
+👤 **Bupilipili**
+
+- GitHub: [@bupilipili](https://github.com/bupilipili)
+- Twitter: [@ErickBupilipili](https://twitter.com/ErickBupilpili)
+- LinkedIn: [ErickBupilipili](https://linkedin.com/in/ErickBupilipili)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Making the site responsive]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [https://github.com/jecode93/awesome-book/issues].
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can star the repository to make it look better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank the entire Microverse community.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Great job team🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.